### ISSUE TITLE
Kernel version (5.3.8-859.native/5.3.11-869.native) can't be found; no boot.

### ISSUE THREAD

**Issue by JSEHV** [clearlinux/distribution#1459]
After updating to version 31520, also a new kernel version (5.3.8-859.native) was installed. This version I was not able to boot, as it mentioned the kernel couldn't be found.

When I reverted the default in the loader config file back to load the previous kernel (5.3.8.854.native) the system booted fine again.

Please check this updated kernel/boot files, as something seems to be wrong.

System specs can be found here: https://github.com/clearlinux/distribution/issues/976#issuecomment-548003334

**Comment by ahkok** [comment]
can you try, and paste the output of:

`sudo systemctl start boot.mount`
`find /boot`

also interesting to try:

`sudo clr-boot-manager update`

**Comment by JSEHV** [comment]
```
sudo systemctl start boot.mount
find /boot
```

```
/boot
/boot/EFI
/boot/EFI/org.clearlinux
/boot/EFI/org.clearlinux/bootloaderx64.efi
/boot/EFI/org.clearlinux/loaderx64.efi
/boot/EFI/org.clearlinux/freestanding-00-intel-ucode.cpio
/boot/EFI/org.clearlinux/freestanding-i915-firmware.cpio.xz
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.8-854
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.8-859
/boot/EFI/BOOT
/boot/EFI/BOOT/BOOTX64.EFI
/boot/loader
/boot/loader/entries
/boot/loader/entries/Clear-linux-native-5.3.8-854.conf
/boot/loader/entries/Clear-linux-native-5.3.8-859.conf
/boot/loader/loader.conf
```

after 

`sudo clr-boot-manager update`

I still have to remove line 'initrd /EFI/org.clearlinux/freestanding-00-intel-ucode.cpio' due to issue #976 

Still no boot at native.5.3.8-859 only at native.5.3.8-854

**Comment by ahkok** [comment]
@dorileo

**Comment by bryteise** [comment]
I wasn't able to replicate this myself. Could you paste the contents of the `/boot/loader/entries/Clear-linux-native-5.3.8-859.conf` file just in case?

**Comment by PieroSimonet** [comment]
Hi I'm having the same problem with this kernel. This is my output if it can help

`sudo find /boot
/boot
/boot/EFI
/boot/EFI/org.clearlinux
/boot/EFI/org.clearlinux/bootloaderx64.efi
/boot/EFI/org.clearlinux/loaderx64.efi
/boot/EFI/org.clearlinux/freestanding-00-intel-ucode.cpio
/boot/EFI/org.clearlinux/freestanding-clr-init.cpio.gz
/boot/EFI/org.clearlinux/freestanding-i915-firmware.cpio.xz
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.8-854
/boot/EFI/org.clearlinux/initrd-org.clearlinux.native.5.3.8-854
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.8-859
/boot/EFI/org.clearlinux/initrd-org.clearlinux.native.5.3.8-859
/boot/EFI/BOOT
/boot/EFI/BOOT/BOOTX64.EFI
/boot/loader
/boot/loader/entries
/boot/loader/entries/Clear-linux-native-5.3.8-854.conf
/boot/loader/entries/Clear-linux-native-5.3.8-859.conf
/boot/loader/loader.conf
`

`sudo cat /boot/loader/entries/Clear-linux-native-5.3.8-859.conf
title Clear Linux OS
linux /EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.8-859
initrd /EFI/org.clearlinux/initrd-org.clearlinux.native.5.3.8-859
initrd /EFI/org.clearlinux/freestanding-00-intel-ucode.cpio
initrd /EFI/org.clearlinux/freestanding-clr-init.cpio.gz
initrd /EFI/org.clearlinux/freestanding-i915-firmware.cpio.xz
options root=UUID=a50285e4-4ba7-4b3f-b80e-191058877dba rd.luks.uuid=b6a9f900-1ba3-416e-a229-1acad4ffe1c6 quiet console=tty0 console=ttyS0,115200n8 cryptomgr.notests init=/usr/bin/initra-desktop initcall_debug intel_iommu=igfx_off kvm-intel.nested=1 no_timer_check noreplace-smp page_alloc.shuffle=1 rcu_nocbs=0-64 rcupdate.rcu_expedited=1 rootfstype=ext4,btrfs,xfs tsc=reliable rw module.sig_unenforce rootflags=x-systemd.device-timeout=0
`

**Comment by puneetse** [comment]
I see between the reported kernel version some patches were added regarding ucode loading and binutils workaround. @jtkukunas @fenrus75 could those changes be causing this?

**Comment by JSEHV** [comment]
Solved in kernel version 5.3.9-863.native

`swupd update --version 31530`

**Comment by JSEHV** [comment]
Same issue is back in Clear-linux-native-5.3.11-869. 
Can't boot this kernel.

```
sudo find /boot
/boot
/boot/EFI
/boot/EFI/org.clearlinux
/boot/EFI/org.clearlinux/bootloaderx64.efi
/boot/EFI/org.clearlinux/loaderx64.efi
/boot/EFI/org.clearlinux/freestanding-00-intel-ucode.cpio
/boot/EFI/org.clearlinux/freestanding-i915-firmware.cpio.xz
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.9-863
/boot/EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.11-869
/boot/EFI/BOOT
/boot/EFI/BOOT/BOOTX64.EFI
/boot/loader
/boot/loader/entries
/boot/loader/entries/Clear-linux-native-5.3.9-863.conf
/boot/loader/entries/Clear-linux-native-5.3.11-869.conf
/boot/loader/loader.conf
```
```
sudo cat /boot/loader/entries/Clear-linux-native-5.3.11-869.conf 
title Clear Linux OS
linux /EFI/org.clearlinux/kernel-org.clearlinux.native.5.3.11-869
initrd /EFI/org.clearlinux/freestanding-i915-firmware.cpio.xz
options root=PARTUUID=4d5cb731-8471-435c-b196-4c0cc7b92b3d quiet console=tty0 console=ttyS0,115200n8 cryptomgr.notests init=/usr/bin/initra-desktop initcall_debug intel_iommu=igfx_off kvm-intel.nested=1 no_timer_check noreplace-smp page_alloc.shuffle=1 rcu_nocbs=0-64 rcupdate.rcu_expedited=1 rootfstype=ext4,btrfs,xfs tsc=reliable rw 

```

**Comment by JSEHV** [comment]
Solved (again) in kernel 5.3.13-872.native

`swupd update --version 31770`
